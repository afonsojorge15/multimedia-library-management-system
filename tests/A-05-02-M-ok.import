USER:Darth Vader:darth@dark.force.com
USER:Luke Skywalker:last.jedi@universe.com
USER:Obi-Wan Kenobi:obiwan@jedi.org
