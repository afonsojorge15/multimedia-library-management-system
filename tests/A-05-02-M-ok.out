Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 0 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
1 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
2 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 